Source/WebCore/platform/mediastream/gstreamer/GStreamerCaptureDeviceManager.h

```diff
@@ -46,8 +47,9 @@ class GStreamerCaptureDeviceManager : public CaptureDeviceManager, public Realti
     ~GStreamerCaptureDeviceManager();
     std::optional<GStreamerCaptureDevice> gstreamerDeviceWithUID(const String&);
 
+    const Vector<CaptureDevice>& speakerDevices();
     const Vector<CaptureDevice>& captureDevices() final;
-    virtual CaptureDevice::DeviceType deviceType() = 0;
+    virtual OptionSet<CaptureDevice::DeviceType> deviceTypes() = 0;
```

Comment:
I can see how `speakerDevices()` and `captureDevices()` aren't `const`, as they trigger `refreshCaptureDevices()`. 

But `deviceTypes()` could probably be `const` as it seems to return a static set

Reply:
That works, thanks for the suggestion 👌🏼